ocaml/default.nix

```diff
@@ -449,12 +453,15 @@ websocketafPackages // {
     propagatedBuildInputs = [ xmlm uri ptime ];
   };
 
+  tyxml = callPackage ./tyxml { ocamlPackages = oself; };
   tyxml-jsx = callPackage ./tyxml/jsx.nix { ocamlPackages = oself; };
   tyxml-ppx = callPackage ./tyxml/ppx.nix { ocamlPackages = oself; };
   tyxml-syntax = callPackage ./tyxml/syntax.nix { ocamlPackages = oself; };
 
   unstrctrd = callPackage ./unstrctrd { ocamlPackages = oself; };
 
+  uri = callPackage ./uri { ocamlPackages = oself; };
```

Comment:
pretty sure URI exists in nixpkgs. what's the reasoning for adding it here?

Reply:
Waiting for this because of a bug: https://github.com/NixOS/nixpkgs/pull/124342